launchable/commands/subset.py

```diff
@@ -131,10 +132,19 @@
     help="outputs the exclude test list. Switch the subset and rest.",
     is_flag=True,
 )
+@click.option(
+    "--non-blocking",
+    "is_non_blocking",
+    help="""
+    Do not wait for subset requests in observation mode.
+    Please note that the CLI does not output a subset list if you specify this option.
+    """,
+    is_flag=True,
```

Comment:
How about defining it as a hidden option? Because this option is experimental now.

Reply:
[a159014](https://github.com/launchableinc/cli/pull/963/commits/a15901455830a97af3e328d3854f7055369f3d5c)